src/components/Badges/Badges.tsx

```diff
@@ -0,0 +1,42 @@
+import { colors } from '../../typography/colors';
+import { Caption } from '../typography/caption/caption';
+import './index.css';
+import { ReactNode, useState } from 'react';
+
+export interface PropsBadges {
+  title: string;
+  icon: ReactNode;
+  onClick: () => void;
+  selected: boolean;
+}
+
+export function Badges({ title, icon, onClick, selected }: PropsBadges) {
+  const [hover, setHover] = useState(false);
+  const mouseEnter = () => {
+    setHover(true);
+  };
+  const mouseLeave = () => {
+    setHover(false);
+  };
+  const handleClick = () => {
+    onClick();
+  };
+
+  const badgesStyle = {
+    border: selected || hover ? 'none' : `1px solid ${colors.NeutralMedium}`,
+    backgroundColor: selected ? colors.CTA : hover ? '#29D6E733' : colors.Neutral,
+  };
+
+  return (
+    <div
+      className="container-badges"
+      onClick={handleClick}
+      style={badgesStyle}
+      onMouseEnter={mouseEnter}
```

Comment:
```suggestion
      onMouseEnter={handleMouseEnter}
```

Outra convenção de nomenclatura.